src/templates/jest-global-setup.js

```diff
@@ -1,11 +1,10 @@
 module.exports = async () => {
+    // Check if the developer has opted out of telemetry and set the environment variable accordingly
+    const optOutTelemetry = process.env.PYTHAGORA_TELEMETRY_ENABLED === 'false';
 
-    // function that sets up Mongo to be used in tests
-    // global.setUpDb = () => {};
+    if (optOutTelemetry) {
+        process.env.PYTHAGORA_TELEMETRY_ENABLED = 'false';
```

Comment:
It is not clear what this line of code does. Why is `PYTHAGORA_TELEMETRY_ENABLED` set to the same value that it already contains?

Reply:
Yes, you're right, it was a redundant if block, i have rectified this too.